src/EditorFeatures/CSharpTest/AddUsing/AddUsingTests.cs

```diff
@@ -6124,5 +6124,82 @@ public class Goo
     }
 }", testHost);
         }
+
+        [WorkItem(1239, @"https://github.com/dotnet/roslyn/issues/1239")]
```

Comment:
these tests moved.  teh standard analyzer supports these once neal did his work to still bind lambdas with syntax errors in them.